tests/unit/ui/components/TestRating.js

```diff
@@ -40,13 +40,18 @@ describe('ui/components/Rating', () => {
   });
 
   it('can be classified as small', () => {
-    const root = render({ styleName: 'small' });
+    const root = render({ styleSize: 'small' });
     expect(root.element.className).toContain('Rating--small');
   });
 
-  it('throws an error for invalid styleNames', () => {
-    expect(() => render({ styleName: 'x-large' }))
-      .toThrowError(/styleName=x-large is not a valid value; possible values: small,/);
+  it('classifies as unowned by default', () => {
+    const root = render();
+    expect(root.element.className).not.toContain('Rating--by-owner');
```

Comment:
Can you also add a test for the presence of `Rating--by-owner`? Or did I miss it?